explainer.md

```diff
@@ -55,28 +55,28 @@ Provide web apps with efficient access to built-in (software and hardware) media
 
 We build on top of [WHATWG Streams](https://streams.spec.whatwg.org/) (in particular [TransformStreams](https://streams.spec.whatwg.org/#ts-class)) and [MediaStreamTracks](https://www.w3.org/TR/mediacapture-streams/#dom-mediastreamtrack).
 
-**EncodedAudioPacket**s and **EncodedVideoFrame**s provide access to codec-specific encoded media bytes so they may be transported, saved, etc.
+**EncodedAudioChunk**s and **EncodedVideoChunk**s provide access to codec-specific encoded media bytes so they may be transported, saved, etc.
 
-**DecodedAudioPacket**s and **DecodedVideoFrame**s provide opaque handles for passing to/from other APIs (such as to/from MediaStreamTracks).   
+**AudioPacket**s and **VideoFrame**s may be passed opaque handles to other APIs (such as to/from MediaStreamTracks). Alternatively, AudioPacket may expose an [AudioBuffer](https://webaudio.github.io/web-audio-api/#audiobuffer) for rendering via [AudioWorklet](https://webaudio.github.io/web-audio-api/#audioworklet), while VideoFrame may expose an ImageBitmap for efficient rendering to canvas.
 
-An **AudioTrackReader** converts a MediaStreamTrack into a ReadableStream of DecodedAudioPacket.
+An **AudioTrackReader** converts a MediaStreamTrack into a ReadableStream of AudioPacket.
 
-An **AudioEncoder** is a TransformStream from DecodeAudioPacket to EncodedAudioPacket.
+An **AudioEncoder** is a TransformStream from AudioPacket to EncodedAudioChunk.
 
-An **AudioDecoder** is a TransformStream from EncodedAudioPacket to DecodedAudioPacket.
+An **AudioDecoder** is a TransformStream from EncodedAudioChunk to AudioPacket.
 
-An **AudioTrackWriter** converts a WritableStream of DecodedAudioPacket into a MediaStreamTrack.
+An **AudioTrackWriter** converts a WritableStream of AudioPacket into a MediaStreamTrack.
 
-A **VideoTrackReader** converts a MediaStreamTrack into a ReadableStream of DecodedVideoFrame.
+A **VideoTrackReader** converts a MediaStreamTrack into a ReadableStream of VideoFrame.
 
-A **VideoEncoder** is a TransformStream from DecodeVideoFrame to EncodedVideoFrame.
+A **VideoEncoder** is a TransformStream from VideoFrame to EncodedVideoChunk.
 
-A **VideoDecoder** is a TransformStream from EncodedVideoFrame to DecodedVideoFrame.
+A **VideoDecoder** is a TransformStream from EncodedVideoChunk to VideoFrame.
 
-A **VideoTrackWriter** converts a WritableStream of DecodedVideoFrame into a MediaStreamTrack.
+A **VideoTrackWriter** converts a WritableStream of VideoFrame into a MediaStreamTrack.
 
 ## Examples
-### Example of decode for low-latency live streaming or cloud gaming 
+### Example of decode and rendering via \*TrackWriters for low-latency live streaming or cloud gaming
```

Comment:
What does the asterisk mean?

Recommend moving below the canvas path.

Reply:
Asterisk just a typo. Move complete.